cmd/misc/main.go

```diff
@@ -151,6 +151,39 @@ func main() {
 		for epoch := opts.StartEpoch; epoch <= opts.EndEpoch; epoch++ {
 			err = exporter.ExportEpoch(epoch, rpcClient)
 
+			if err != nil {
+				logrus.Errorf("error exporting epoch: %v", err)
+			}
+			logrus.Printf("finished export for epoch %v", epoch)
+		}
+	case "export-epoch-missed-slots":
+		logrus.Infof("exporting epochs with missed slots")
+		epochs := []uint64{}
+		err = db.ReaderDb.Select(&epochs, `
+			WITH last_exported_epoch AS (
+				SELECT (MAX(epoch)*$1) AS slot 
+				FROM epochs 
+				WHERE finalized 
+				AND rewards_exported
```

Comment:
what is the reason to filter on `rewards_exported` ?

Reply:
To get the last epoch that was exported. If the epoch has not yet been exported we don't need to re-trigger an export.